client/stylesheets/_modal.scss

```diff
@@ -1083,7 +1099,8 @@ div[class="sp-replacer sp-light full-spectrum sp-active"] {
 	border-radius: 4px !important;
 }
 
-.modFeedbackInitial button[data-target="#feedbackBuyerUnsuccessfulModal"]:hover, .modFeedbackInitial button[data-target="#feedbackBuyerNeverModal"]:hover {
+.modFeedbackInitial button[data-target="#feedbackBuyerUnsuccessfulModal"]:hover, 
```

Comment:
Avoid qualifying attribute selectors with an element.<br>Selector `modFeedbackInitial` should be written in lowercase with hyphens<br>Line contains trailing whitespace